addons/event/views/event_views.xml

```diff
@@ -340,51 +340,44 @@
-                                <div class="o_event_right d-flex flex-column justify-content-between">
+                                <div class="o_event_right d-flex flex-column justify-content-between pt-3">
                                     <div>
                                         <h1 class="o_kanban_record_title"><field name="name"/></h1>
-                                        <p t-esc="record.address_id.value"/>
+                                        <div><i class="fa fa-map-marker" title="Location"/> <span class="o_text_overflow" t-esc="record.address_id.value" style="max-width: 85%!important"/></div>
+                                        <div t-if="moment(record.date_begin.raw_value).format('MMM D') !== moment(record.date_end.raw_value).format('MMM D')">
+                                            <i class="fa fa-arrow-right" title="End date"/>
+                                            <t t-esc="moment(record.date_end.raw_value).format('MMM D')"/>
+                                         </div>
                                     </div>
-                                    <div class="o_event_duration_str">Duration <field name="duration_str"/></div>
-                                    <div class="o_event_stat text-left d-flex justify-content-between">
-                                        <div class="text-left inline-block mr-2">
-                                            <button
-                                                type="action"
-                                                class="btn btn-link p-0"
-                                                name="%(event_registration_action_tree)d"
-                                                t-esc="record.seats_expected.raw_value"/>
-                                            <p>Expected</p>
-                                        </div>
-                                        <div class="text-left inline-block">
-                                            <button
-                                                type="action"
-                                                class="btn btn-link my-0 p-0"
-                                                name="%(event_registration_action_tree)d"
-                                                context="{'search_default_confirmed': True}"
-                                                t-esc="record.seats_reserved.raw_value"/>
-                                            <p>Confirmed</p>
-                                        </div>
+                                    <h5 class="o_event_stat">
+                                        <a name="%(act_event_registration_from_event)d" type="action">
+                                            <t t-esc="record.seats_expected.raw_value"/> Expected attendees
+                                        </a>
+                                        <t t-if="(record.seats_reserved.raw_value + record.seats_used.raw_value) > 0 "><br/>
```

Comment:
Make a variable with that?
`<t t-set="total_seats" t-value="record.seats_reserved.raw_value + record.seats_used.raw_value" />`